### PR TITLE
 Fix build with cmake 3.13

### PR DESCRIPTION
**Pull request by xantares**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 2.8)
```
[the remark this answers is not in the document]
Comment (xantares) [reply CMakeLists.txt]: ok

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
 
+if (NOT DEFINED CMAKE_BUILD_TYPE)
+  set (CMAKE_BUILD_TYPE Release CACHE STRING "Build type")
+endif ()
```
[the remark this answers is not in the document]
Comment (xantares) [reply CMakeLists.txt]: ok

**Comment by xantares** [comment]
ping @lrineau

**Comment by lrineau** [comment]
I have approved your newest changes. But I am not the official maintainer of `cgal-swig-bindings`. The final approval will be by @sloriot.

File: CMakeLists.txt [hunk 3]
```diff
@@ -12,16 +12,21 @@ if (POLICY CMP0042)
...
-include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/SWIG_CGAL_Macros.cmake)
+include (SWIG_CGAL_Macros)
 
 find_package(CGAL QUIET COMPONENTS ImageIO)
 find_package(SWIG REQUIRED)
 option( BUILD_PYTHON "Build Python bindings" ON )
 option( BUILD_JAVA "Build Java bindings" ON )
 #set it to OFF by default while it is not fully tested and thus officially supported
 option( BUILD_RUBY "Build Ruby bindings" OFF )
+option( BUILD_SHARED_LIBS "Build shared libraries" ON )
```
Comment (sloriot) [inline CMakeLists.txt]: could you explain this change?

File: cmake/Modules/SWIG_CGAL_Macros.cmake [hunk 4]
```diff
@@ -1,10 +1,12 @@
+include (TargetLinkLibrariesWithDynamicLookup)
```
Comment (sloriot) [inline cmake/Modules/SWIG_CGAL_Macros.cmake]: Could you explain why is this needed now?

Comment (xantares) [reply cmake/Modules/SWIG_CGAL_Macros.cmake]: I removed it, this is unrelated

File: CMakeLists.txt (hunk 3 above)
Comment (xantares) [reply CMakeLists.txt]: I remove it to only include cmake 3.13 changes.

**Comment by sloriot** [comment]
Thanks for the PR, I've merged it. You can removed your branch now. Sorry for the delay.
